src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -475,6 +475,7 @@ impl VirtSandbox {
             path_on_host: image_path.display().to_string(),
             is_readonly: true,
             driver_option: block_driver.clone(),
+            blkdev_aio: BlockDeviceAio::Threads,
```

Comment:
why hard coding  the "threads"  here? Was it better to get it from hypervisor's config?